docs/ImportONNXDefs.md

```diff
@@ -34,10 +34,10 @@ Several tables are defined at the beginning of the script:
 6. `custom_builder_ops_list`: list of operations which need custom build methods to deduce result types
 
 ## Version of Operations
-As stated previous, we try to support the latest version of ONNX operations. The version of each operation currently supported is recorded in gen_onnx_mlir.py. This mechanism provides some stability in version. To check the changes in version, run gen_onnx_mlir.py with flag "--check-version" and the changes will be reported. To move to a newer version, manually update the version dictionary in the script.
+As stated previous, we try to support the latest version of ONNX operations. The version of each operation currently supported is recorded in [utils/gen_onnx_mlir.py](../utils/gen_onnx_mlir.py). This mechanism provides some stability in version. To check the changes in version, run gen_onnx_mlir.py with flag "--check-version" and the changes will be reported. To move to a newer version, manually update the version dictionary in the script.
 
 ### Support Multiple versions
-To support multiple versions of an op, the selected version should be added in the version dictionary in gen_onnx_mlir.py. For example, there are two versions (opset), 11 and 13, forReduceSum is supported. The corresponding entry in version_dic is `'ReduceSum': [13, 11]`.
+To support multiple versions of an op, the selected version should be added in the version dictionary in [utils/gen_onnx_mlir.py](../utils/gen_onnx_mlir.py). For example, there are two versions (opset), 11 and 13, forReduceSum is supported. The corresponding entry in version_dic is `'ReduceSum': [13, 11]`.
```

Comment:
`forReduceSum is supported.` -> `forReduceSum that are supported.`

Reply:
tx